[bl0942] Update docs for accuracy fixes in esphome/esphome#7428 (#4237)
OttoWinter/esphomedocs: components/sensor/bl0942.rst

    sensor:
      - platform: bl0942
        uart_id: uart_bus

>         reset: false

Reviewer: This example needs to be cleaned up a lot (i'll do this in another PR)
```suggestion
```

  to use multiple UART buses.
- **line_frequency** (*Optional*, string): The nominal AC line frequency of the supply voltage. One of ``50Hz``, ``60Hz``. Defaults to ``50Hz``.
- **address** (*Optional*, int): The address of the BL0942 from its strapping pins. Defaults to ``0``.

> - **reset** (*Optional*, boolean): Whether to reset the BL0942 chip on startup. Defaults to ``true`` which resets the charge counter to zero and potentially loses data.

Reviewer: ```suggestion
- **reset** (*Optional*, boolean): Whether to reset the BL0942 chip on startup, resetting all internal counters. Defaults to ``true``.
```
